### DISCUSSION TITLE
Freezed program looking for __init__.py in my original user directory

### DISCUSSION THREAD

**Discussion by morg-mov** [marcelotduarte/cx_Freeze#1961] [Q&A] [answered]
I'm building a program that utilizes a fork of the Discord.py library called Pycord.

I'm trying to freeze it for ease of use for others, but for one reason or another, after testing the frozen program on a workstation other than my own, it throws this runtime warning:
`C:\Users\Morg\<redacted>\venv\Lib\site-packages\discord\__init__.py:22: RuntimeWarning: Package is not installed, and setuptools_scm is not installed. As a fallback, discord._version.__version__ will be set to 0.0.0`

Implying that it's looking for ``__init__.py`` in my python virtual environment that I use for development rather than the included one with the frozen program.

What could be causing this?

**Comment by marcelotduarte** [comment]
This is only debugging information. You can change this using [replace_paths](https://cx-freeze.readthedocs.io/en/latest/setup_script.html#cmdoption-arg-replace_paths).
You can use like this [sample](https://github.com/marcelotduarte/cx_Freeze/blob/main/samples/sqlite/setup.py#L28).

**Reply by morg-mov** [reply]
A similar message appears this time after building, only this time It just has a relative path:
`discord\__init__.py:22: RuntimeWarning: Package is not installed, and setuptools_scm is not installed. As a fallback, discord._version.__version__ will be set to 0.0.0`

Is this a message I can safely ignore, and if so, is it possible to disable it?
By the way, here's my setup script if you want to look at it for yourself:

```
from cx_Freeze import setup, Executable

# Dependencies are automatically detected, but it might need
# fine tuning.
build_options = {'packages': [], 'excludes': [], "replace_paths": [("*", "")]}

base = 'console'

executables = [
    Executable('OSCCord.py', base=base, icon="assets/program-icon/OSCCord_ICO.ico")
]

setup(name='OSCCord',
      version = '1.0.0',
      description = "A program for VRChat's OSC that outputs Discord Messages to your chatbox",
      options = {'build_exe': build_options},
      url = "https://github.com/Morg-S9/VRC-OSCCord",
      maintainer = "morg.mov",
      executables = executables
      )
```

**Reply by marcelotduarte** [reply, marked as answer]
To disable it temporarily, check: https://docs.python.org/3/library/warnings.html#temporarily-suppressing-warnings

I can make a hook to fix this getting the correct version metadata. In this case, open an issue.

**Reply by morg-mov** [reply]
Will do, thanks for your help!
